### ISSUE TITLE
Update logo on "stable" documentation page

### ISSUE THREAD

**Issue by kdpsingh** [kdpsingh/Tidier.jl#74]
The "stable" docs continue to show the old logo even though the "dev" docs show the updated logo. We should fix this.

I think we may need to directly check out the `gh-pages` branch and make edits there. I don't think it's possible to fix it upstream of the GitHub action but I'm not 100% sure.

@pdimens or @lazarusA, do you happen to know?

**Comment by kdpsingh** [comment]
Oh I didn't realize. I just assumed "stable" pointed to the latest released version. I can swap it out.

I wonder if the fact it's pointing to 0.5 is intentional (something to do with Documenter.jl) vs. something I have mis-configured that isn't updating the docs after release.

**Comment by pdimens** [comment]
I use Docusaurus instead of Documenter.jl so I can't provide any wisdom on that point, maybe @lazarusA might know. It would make sense for stable to match the latest release though.

**Comment by kdpsingh** [comment]
Actually I think I know what it is. Since v0.5.0, I fixed the RegistratorBot so that it automatically creates a release on GitHub after the commit is merged with the Julia registry (as opposed to me manually having to do it). My guess is that this workflow isn't triggering the GitHub action to update the Stable release.

**Comment by kdpsingh** [comment]
Are any of you experienced with GitHub actions to help figure out where a hook isn't triggering but should be? @rdboyes @pitmonticone 

I think this is why the "stable" documentation isn't updating following each release. I used to manually tag releases but now it's automated by the RegistratorBot or TagBot.

**Comment by pitmonticone** [comment]
Hi @kdpsingh, are you using [SSH deploy keys](https://github.com/JuliaRegistries/TagBot#ssh-deploy-keys)? 

I'm asking because I know that the default TagBot setup that uses the `GITHUB_TOKEN` doesn't have the permissions to trigger docs builds (see [here](https://github.com/JuliaDocs/Documenter.jl/issues/1277) for further details).

**Comment by kdpsingh** [comment]
I don't think so. Will read more about this.

What's weird is that up until v0.5.0, I was manually creating the release in GitHub and that was triggering the stable docs build just fine.

But I believe you're right - if TagBot doesn't have permission to run this process, that's probably why stable docs aren't updating.

**Comment by pitmonticone** [comment]
Yes, I tend to believe that if you create the [SSH key](https://developer.github.com/v3/guides/managing-deploy-keys/#setup-2) with write permission and the associated [repository secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets#creating-encrypted-secrets) everything should work as expected.
